save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/TestSuitesController.kt

```diff
@@ -60,20 +61,24 @@ class TestSuitesController(
         val repos = ClassPathResource(configProperties.reposFileName)
             .file
             .readText()
-            .split("\n")
-        log.info("Will update test suites from ${repos.size} repos")
-        repos
-            .map { GitDto(it) }
-            .forEach {
-                log.info("Update test suite from ${it.url}")
-                preprocessorWebClient
-                    .post()
-                    .uri("/uploadStandardTestSuite")
-                    .body(BodyInserters.fromValue(it))
-                    .retrieve()
-                    .toBodilessEntity()
-                    .subscribe()
+            .lines()
+            .map {
+                val splitRow = it.split("\\s".toRegex())
+                require(splitRow.size == 2)
+                TestSuiteRepo(splitRow.first(), splitRow[1].split(";"))
             }
+        log.info("Will update test suites from ${repos.size} repos")
+        Flux.fromIterable(repos).flatMap {
+            log.info("Update test suite from ${it.gitUrl}")
+            preprocessorWebClient
+                .post()
+                .uri("/uploadStandardTestSuite")
+                .body(BodyInserters.fromValue(it))
+                .retrieve()
+                .toBodilessEntity()
+        }
+            .subscribe()
```

Comment:
I believe we should wait for this operation to finish before we return status OK

Reply:
Actually, let's move this logic into preprocessor and have a simple endpoint in backend, which will send request to preprocessor